frontend/public/components/operator-lifecycle-manager/_operator-lifecycle-manager.scss

```diff
@@ -42,6 +42,11 @@
   font-size: 13px;
 }
 
+.co-clusterserviceversion-empty__state__namespace {
```

Comment:
I don't think we should be special casing one specific empty state message. I know this is probably what was in the design, but we should be consistent across pages.